### PR TITLE
Update and rename udev/Sony Interactive Entertainment Wireless Controller.cfg to udev/Sony Interactive Entertainment DualSense Wireless Controller.cfg

### PR DESCRIPTION
**Pull request by davidhedlund**
Changed L2 and R2. My controller uses the latest DualSense firmware at this writing (0402). 

I mentioned the firmware version in the file to avoid confusion.

See https://github.com/libretro/retroarch-joypad-autoconfig/issues/899 for more information.

### CODE REVIEW

**Comment by davidhedlund** [comment]
@RobLoach This is 100% correct now. You can merge it.

File: udev/Sony Interactive Entertainment DualSense Wireless Controller.cfg [hunk 1]
```diff
@@ -16,8 +23,8 @@ input_a_btn = "1"
 input_x_btn = "2"
 input_l_btn = "4"
 input_r_btn = "5"
-input_l2_btn = "6"
-input_r2_btn = "7"
+input_l2_axis = "6"
+input_r2_axis = "7"
```
Comment (RobLoach) [inline udev/Sony Interactive Entertainment DualSense Wireless Controller.cfg:27]: The majority of other `_axis` inputs use a `+2` or similar. Sure these are correct? Thanks.

Comment (davidhedlund) [reply udev/Sony Interactive Entertainment DualSense Wireless Controller.cfg:27]: Yes, I triple checked. I checked a 4th time right now when you asked about this, and it's still correct. :)

**Comment by davidhedlund** [comment]
Excellent! I bought this controller 1.5 years ago. It's my primary controller, and now a 100% correct autoconfig is finally uploaded.

File: udev/Sony Interactive Entertainment DualSense Wireless Controller.cfg (hunk 1 above)
Comment (davidhedlund) [reply udev/Sony Interactive Entertainment DualSense Wireless Controller.cfg:27]: > The majority of other `_axis` inputs use a `+2` or similar. Sure these are correct? Thanks.

Aftermath:

My PR was been reverted due to a bug in RetroArch:
* https://github.com/libretro/retroarch-joypad-autoconfig/pull/1135/files

```
# Note 2024-06-04 :
#   See https://github.com/libretro/RetroArch/issues/6920
#   RetroArch's in-app feature to create autoconfig files is flawed and will attempt to create the 2 lines :
#     input_l2_btn = "6"
#     input_r2_btn = "7"
#   Those 2 lines are a downgrade from the 2 following lines which additionally allow l2/r2 to be polled as analog axises :
input_l2_axis = "+2"
input_r2_axis = "+5"
```

**Pull request merged**